assets/js/modules/analytics/components/common/AccountCreate/index.js

```diff
@@ -25,28 +25,32 @@ import { useCallback, useState, useEffect } from '@wordpress/element';
 /**
  * Internal dependencies
  */
+import Data from 'googlesitekit-data';
+import { STORE_NAME, FORM_ACCOUNT_CREATE, PROVISIONING_SCOPE } from '../../../datastore/constants';
+import { CORE_SITE } from '../../../../../googlesitekit/datastore/site/constants';
+import { CORE_USER } from '../../../../../googlesitekit/datastore/user/constants';
+import { CORE_FORMS } from '../../../../../googlesitekit/datastore/forms/constants';
+import { CORE_LOCATION } from '../../../../../googlesitekit/datastore/location/constants';
+import { ERROR_CODE_MISSING_REQUIRED_SCOPE } from '../../../../../util/errors';
+import { trackEvent } from '../../../../../util';
+import { getAccountDefaults } from '../../../util/account';
+import { isFeatureEnabled } from '../../../../../features';
 import Button from '../../../../../components/Button';
 import Link from '../../../../../components/Link';
 import ProgressBar from '../../../../../components/ProgressBar';
-import { trackEvent } from '../../../../../util';
-import { ERROR_CODE_MISSING_REQUIRED_SCOPE } from '../../../../../util/errors';
+import StoreErrorNotices from '../../../../../components/StoreErrorNotices';
+import GA4Notice from '../GA4Notice';
+import GA4PropertyNotice from '../GA4PropertyNotice';
 import TimezoneSelect from './TimezoneSelect';
 import AccountField from './AccountField';
 import PropertyField from './PropertyField';
 import ProfileField from './ProfileField';
 import CountrySelect from './CountrySelect';
-import StoreErrorNotices from '../../../../../components/StoreErrorNotices';
-import GA4Notice from '../GA4Notice';
-import { STORE_NAME, FORM_ACCOUNT_CREATE, PROVISIONING_SCOPE } from '../../../datastore/constants';
-import { CORE_SITE } from '../../../../../googlesitekit/datastore/site/constants';
-import { CORE_USER } from '../../../../../googlesitekit/datastore/user/constants';
-import { CORE_FORMS } from '../../../../../googlesitekit/datastore/forms/constants';
-import { CORE_LOCATION } from '../../../../../googlesitekit/datastore/location/constants';
-import { getAccountDefaults } from '../../../util/account';
-import Data from 'googlesitekit-data';
 const { useDispatch, useSelect } = Data;
 
 export default function AccountCreate() {
+	const isGA4enabled = isFeatureEnabled( 'ga4setup' );
```

Comment:
This should use `useFeature` instead. `isFeatureEnabled` is only intended for use outside of React.

Reply:
This is needed to make sure the storybook story works correctly.

Reply:
Sounds good. That's fine for now; we'll take a closer look in QA to not block things now 👍